app/views/application_templates/show.haml

```diff
@@ -14,13 +14,27 @@
       = button_to 'Activate application',
           toggle_active_application_url(@template)
 
-- if @current_user.try(:staff?) || @template.active?
+- if @template.active?
   .form
     %h1.title Application for #{@template.position.name_and_department}
     %strong
       Fields marked with an asterisk (
       %span.ast *
       ) are required.
+    - if should_allow_form_refilling? && @old_applications.present?
+      .row#old_application_block
+        .col-md-8.col-md-offset-2#subtitle
+          You have some old applications you can take responses from!
+        %table.col-md-8.col-md-offset-2
+          %tr
+            %th
+              Submitted
+          - @old_applications.each do |app|
+            %tr
+              %td
+                = format_date_time app.created_at
+              %td.load_column
+                = link_to 'Load reponses', url_for(params.merge(:load_id => app.id))
```

Comment:
Is it possible to pass the template directly in since this is the page with the new `/department/position` routing?